### ISSUE TITLE
is it possible to change the learning rate ?

### ISSUE THREAD

**Issue by pmejane** [Hironsan/anago#80]
When I train the model, the loss increase. I would like to change the learning rate but I don't see it in the code.

**Comment by Hironsan** [comment]
You can pass optimizer to `Sequence` class.

```python
from keras import optimizers

sgd = optimizers.SGD(lr=0.01, decay=1e-6, momentum=0.9)
model = anago.Sequence(..., optimizer=sgd)
```

**Comment by pmejane** [comment]
Great ! Thanks a lot
